### PR TITLE
validation fixes and updates

### PR DESCRIPTION
**Pull request by manjuX**

### CODE REVIEW

File: tutorials/btp-app-cap-mta-deployment/btp-app-cap-mta-deployment.md [hunk 1]
```diff
@@ -331,7 +336,7 @@ Alternatively, role collections can be manually assigned in the SAP BTP cockpit.
...
-> `NPM` uses a file called `package-lock.json` to remember which actual version of package was installed and later installs these exact versions and ignores any updates in minor releases not explicitly specified in the package.json file. Maintaining this is important for production applications for consistency. For the purposes of this tutorial we would like you to use the latest versions of the packages. To ensure this, delete your older `package-lock.json` and run `npm install --package-lock-only` to generate it again to avoid errors due to use of older versions.
+> `NPM` uses a file called `package-lock.json` to remember which actual version of package was installed and later installs these exact versions and ignores any updates in minor releases not explicitly specified in the `package.json` file. Maintaining this is important for production applications for consistency. For the purposes of this tutorial we would like you to use the latest versions of the packages. To ensure this, delete your older `package-lock.json` and run `npm install --package-lock-only` to generate it again to avoid errors due to use of older versions.
```
Comment (MichaelCzcz) [inline tutorials/btp-app-cap-mta-deployment/btp-app-cap-mta-deployment.md:339]: Please do not say "we" when addressing the user. Say "you" instead.

File: tutorials/btp-app-ci-cd-btp/btp-app-ci-cd-btp.md [hunk 2]
```diff
@@ -259,7 +255,9 @@ If your GitHub repository is not private, you can skip this section. If your Git
 
 13. Leave the execution of the **Additional Unit Tests** switched off.
 
-14. Leave **Acceptance Stage** switched off.
+14. Leave **Acceptance** switched off.
+
+15. Leave **SonarQube Scan** under **Compliance** switched off.
```
Comment (MichaelCzcz) [inline tutorials/btp-app-ci-cd-btp/btp-app-ci-cd-btp.md:260]: CHECK SPELLING:
  
  >Error:
      file: btp-app-ci-cd-btp.md
      line: 260
      reason: SonarQube

use back-ticks

File: tutorials/btp-app-hana-cloud-setup/btp-app-hana-cloud-setup.md [hunk 3]
```diff
@@ -263,9 +262,9 @@ If you need to create an SAP HANA Cloud service instance first &rarr; continue w
     > ```
 
 
-[VALIDATE_1]
 [OPTION END]
 
+[VALIDATE_1]
```
Comment (MichaelCzcz) [inline tutorials/btp-app-hana-cloud-setup/btp-app-hana-cloud-setup.md:267]: This code is obsolete. Please use v2 parser syntax.

File: tutorials/btp-app-prepare-btp/btp-app-prepare-btp.md [hunk 4]
```diff
@@ -56,9 +57,9 @@ Continue with **Step 8: Check and assign entitlements** below.
 [ACCORDION-BEGIN [Step 3: ](Create a Live Account)]
 If you choose to create an account on Live, you have to select a number of services that you need to subscribe to, for example, an SAP HANA database. For each service, there are so-called `entitlements`, which are basically the service plans and the number of units that you want from each service. When you create an account, you need to provide these also.
 
-[VALIDATE_1]
 
 
+[VALIDATE_1]
```
Comment (MichaelCzcz) [inline tutorials/btp-app-prepare-btp/btp-app-prepare-btp.md:62]: This code is obsolete. Please use v2 parser syntax.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/btp-app-cap-mta-deployment/btp-app-cap-mta-deployment.md (hunk 1 above)
Comment (manjuX) [reply tutorials/btp-app-cap-mta-deployment/btp-app-cap-mta-deployment.md:339]: 'we' in this paragraph is not referring to the user, the user is referred to as you.

File: tutorials/btp-app-ci-cd-btp/btp-app-ci-cd-btp.md (hunk 2 above)
Comment (manjuX) [reply tutorials/btp-app-ci-cd-btp/btp-app-ci-cd-btp.md]: fixed.

**Comment by manjuX** [comment]
Hi Michael,

We're authoring our tutorials in MkDocs and have a tool that converts our syntax to v1. We did not update our tool based on the understanding that v1 will still be supported.

For more context on this, you can take a look at the discussion in this PR https://github.com/sap-tutorials/Tutorials/pull/20753.

Thanks for your help!

Best regards,
Manju

File: tutorials/btp-app-prepare-btp/btp-app-prepare-btp.md (hunk 4 above)
Comment (manjuX) [reply tutorials/btp-app-prepare-btp/btp-app-prepare-btp.md:62]: please see comment.

File: tutorials/btp-app-hana-cloud-setup/btp-app-hana-cloud-setup.md (hunk 3 above)
Comment (manjuX) [reply tutorials/btp-app-hana-cloud-setup/btp-app-hana-cloud-setup.md:267]: please see comment.

**Pull request merged**